presentation/workshop.org

```diff
@@ -0,0 +1,44 @@
+#+REVEAL_ROOT: https://cdn.jsdelivr.net/reveal.js/3.0.0/
+# #+REVEAL_TITLE_SLIDE: <h1>%t</h1><h2>%a</h2><h4>Queensland&nbsp;Functional&nbsp;Programming&nbsp;Lab</h4><h3>%e</h3>
+
+#+OPTIONS: num:nil
+#+OPTIONS: toc:nil
+
+#+TITLE: Property-based state machine testing
+#+AUTHOR: Sean Chalmers, Jack Kelly
+#+EMAIL:
+
+* Workshop
+  https://github.com/qfpl/state-machine-testing-course
+  
+* Command
+  #+BEGIN_SRC haskell
+    data Command n m (state :: (* -> *) -> *) = forall input output.
```

Comment:
Alpha-convert `n -> g`, add comment "g is often MonadGen, m often MonadTest, MonadIO"